sw/device/tock/chips/earlgrey/src/lib.rs

```diff
@@ -0,0 +1,16 @@
+//! Drivers and chip support for the Ibex soft core.
+
+#![feature(asm, concat_idents, const_fn, naked_functions)]
+#![feature(in_band_lifetimes)]
```

Comment:
Consider not using this feature, since it's not mainline Rust and I'm working to remove it from Tock.